tests/store/advocates.spec.ts

```diff
@@ -0,0 +1,153 @@
+import advocatesModule from '~/store/modules/advocates'
+
+const advocate1 = {
+  city: 'Lima',
+  country: 'Peru',
+  image: 'https://example.com/img/1.jpg',
+  name: 'John Doe',
+  region: 'South America'
+}
+const advocate2 = {
+  city: 'Munich',
+  country: 'Germany',
+  image: 'https://example.com/img/2.jpg',
+  name: 'Max Mustermann',
+  region: 'Europe'
+}
+
+/**
+ * ACTIONS
+ * -----------------------------------------------------------------------------
+ */
+
+describe('fetchAdvocates', () => {
+  it('commits "setAdvocates" with a list of fetched elements', async () => {
+    const commit = jest.fn()
+
+    const fetchAdvocates = advocatesModule.actions.fetchAdvocates as Function
+    await fetchAdvocates({ commit })
+
+    expect(commit).toHaveBeenCalledWith(
+      'setAdvocates',
+      expect.arrayContaining([expect.anything()])
+    )
```

Comment:
As discussed offline, it seems to be very tricky to _test the right thing_ in this case.

Since this action wasn't introduced with this PR, I'll remove the test for now. Better to still have no test than to introduce a bad / misleading test.